theme/js/components/vue/preview.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="modal-wrapper">
```

Comment:
Can't we abstract `modal-*` blocks with slots and stuff? Maybe it's overkill.

Reply:
We can absolutely do that, I've felt it was overkill at the moment since we only have one modal and the "template" is only 3 divs. My idea was to keep it simple until the need for more abstraction arises and refactor at this point but we can do it earlier if you'd prefer